{"timestamp": "2025-10-13T00:34:09.155236", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
{"timestamp": "2025-10-13T00:34:09.155720", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
{"timestamp": "2025-10-13T00:34:09.155985", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
{"timestamp": "2025-10-13T00:34:09.163156", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
{"timestamp": "2025-10-13T00:34:09.163948", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
{"timestamp": "2025-10-13T00:34:09.182488", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
{"timestamp": "2025-10-13T00:34:09.185087", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
{"timestamp": "2025-10-13T00:34:09.193571", "level": "WARNING", "logger": "app.core.database", "message": "Redis connection failed: Error 61 connecting to localhost:6379. Connection refused.", "module": "database", "function": "get_redis_client", "line": 99, "taskName": null}
